src/Sql/dbo/Stored Procedures/SsoConfig_ReadByOrganizationId.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE PROCEDURE [dbo].[SsoConfig_ReadByOrganizationId]
+@OrganizationId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
```

Comment:
Add `TOP 1` here as well. In the future we may have multiple based on a type, but for now it should only ever be 1 record per organization.